<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::RemoteVideoSetting</class>
 <widget class="QWidget" name="ns::RemoteVideoSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>205</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>263</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}
QSlider::groove:horizontal {
    
    height: 5px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #000000, stop:1 #ffffff);
 	margin: 5px 0;
	border-radius: 3px;
}

QSlider::handle:horizontal {   
    width: 7px;
	height: 13px;
   	margin: -5px 0;
	border-image: url(:/images/ui/vs_sliderblock.png);
}

#widgetTop {
	background: rgba(24, 47, 64, 90%);
}

#widgetTop #pushButtonClose {
	border-image: url(:/images/ui/caption_close.png);
}

#widgetTop #pushButtonClose:hover{
	border-image: url(:/images/ui/caption_close_hover.png);
}

#widgetTop #pushButtonClose:pressed {
	border-image: url(:/images/ui/caption_close_pressed.png);
}


#widgetBrightness{
	background: rgba(16, 34, 42, 75%);
}

#widgetBrightness #labelSliderIcon {
	border-image: url(:/images/ui/vs_brightness.png);
}

#widgetProportion{
	background: rgba(20, 39, 53, 80%);
}

#widgetRotation {
	background: rgba(16, 31, 42, 75%);
}

#widgetRotation #pushButtonStraight {
	border-image: url(:/images/ui/vs_straight.png);
}

#widgetRotation #pushButtonStraight:hover {
	border-image: url(:/images/ui/vs_straight_hover.png);
}

#widgetRotation #pushButtonStraight:pressed {
	border-image: url(:/images/ui/vs_straight_pressed.png);
}

#widgetRotation #pushButtonRightangle {
	border-image: url(:/images/ui/vs_rightangel.png);
}

#widgetRotation #pushButtonRightangle:hover {
	border-image: url(:/images/ui/vs_rightangel_hover.png);
}

#widgetRotation #pushButtonRightangle:pressed {
	border-image: url(:/images/ui/vs_rightangel_pressed.png);
}

QRadioButton::indicator::unchecked {
     image: url(:/images/ui/vs_radio_disable.png);
}

QRadioButton::indicator::unchecked:hover {
     image: url(:/images/ui/vs_radio_disable_hover.png);
}

QRadioButton::indicator::unchecked:pressed {
     image: url(:/images/ui/vs_radio_disable_pressed.png);
}

QRadioButton::indicator::checked {
     image: url(:/images/ui/vs_radio_enable.png);
}



QLabel{
	color: #FFFFFF;
	font-size: 11px;
}

QRadioButton {
	color: #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTittle">
        <property name="text">
         <string>Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>462</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sysPanelContainer">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sysPanel" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>36</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBrightness" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelBrightness">
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetBrightnessSliderContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelSliderIcon">
           <property name="minimumSize">
            <size>
             <width>11</width>
             <height>11</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>11</width>
             <height>11</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="minimum">
            <number>-128</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="sliderPosition">
            <number>80</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetProportion" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="labelProportion">
        <property name="text">
         <string>Proportion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetProportionRadioContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonNormal">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Normal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPorprotion1">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>16:9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonProportion2">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>4:3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCovered">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Covered</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetRotation" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="labelRotation">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetRotationButtonContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonStraight">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRightangle">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widgetProportion</zorder>
  <zorder>widgetRotation</zorder>
  <zorder>widgetTop</zorder>
  <zorder>widgetBrightness</zorder>
 </widget>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ns::RemoteVideoSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
